### DISCUSSION TITLE
HQ Video recording?

### DISCUSSION THREAD

**Discussion by jadasse** [ScratchAddons/ScratchAddons#5137] [Ideas] [answered]
The video recording quality is very bad, the VP8 encoding is very lossy, and the frame rate and duration are not handled correctly. So, do you think there could be a higher quality encoding like AVC or VP8 (or VP9 if implemented)-improved quality of video recording? Also, can I change the resolution of the recording like 1080p?

**Comment by BroJac5246** [comment, marked as answer]
Feel free to create an issue. I just want to point out, however, that #5101 has been discussing file sizes, so that's something to keep in mind.

**Reply by jadasse** [reply]
#5139 done,
